src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -33,24 +33,50 @@ import org.dia.core.{SciSparkContext, SciTensor}
  */
 class GTGRunner(val masterURL: String,
                 val paths: String,
-                val partitions: Int) {
```

Comment:
Added in new commits.
There is no reference to the string "ch4" in GTGRunner and it uses the variableName parameter.
GTG operates on a single variable so lets stick with that variable name.
This assumes then that the Netcdf Files being used have longitude and lattitude variable arrays.